crypto/src/mls/mod.rs

```diff
@@ -536,19 +531,23 @@ impl MlsCentral {
     /// * `message` - the encrypted message as a byte array
     ///
     /// # Return type
-    /// A TLS serialized byte array of the `PublicGroupState`
+    /// A TLS serialized byte array of the `GroupInfo`
     ///
     /// # Errors
     /// If the conversation can't be found, an error will be returned. Other errors are originating
     /// from OpenMls and serialization
-    pub async fn export_public_group_state(&mut self, conversation_id: &ConversationId) -> CryptoResult<Vec<u8>> {
+    pub async fn export_group_info(&mut self, conversation_id: &ConversationId) -> CryptoResult<Vec<u8>> {
         let conversation = self.get_conversation(conversation_id).await?;
+        let conversation_lock = conversation.read().await;
+        let group = &conversation_lock.group;
+        let cs = group.ciphersuite();
+        let ct = group.own_leaf().unwrap().credential().credential_type();
```

Comment:
This unwrap is safe (because you can only use this method if you're a member of the group) but we will need to clean it up at some point.